packages/contracts-bedrock/contracts/L1/L2OutputOracle.sol

```diff
@@ -93,49 +93,54 @@ contract L2OutputOracle is OwnableUpgradeable, Semver {
      * @custom:semver 0.0.1
      *
      * @param _submissionInterval    Interval in blocks at which checkpoints must be submitted.
-     * @param _genesisL2Output       The initial L2 output of the L2 chain.
+     * @param _l2BlockTime           The time per L2 block, in seconds.
+     * @param _startingL2Output      The initial L2 output of the L2 chain.
      * @param _startingBlockNumber   The number of the first L2 block.
      * @param _startingTimestamp     The timestamp of the first L2 block.
-     * @param _l2BlockTime           The time per L2 block, in seconds.
      * @param _proposer              The address of the proposer.
      * @param _owner                 The address of the owner.
      */
     constructor(
         uint256 _submissionInterval,
-        bytes32 _genesisL2Output,
+        uint256 _l2BlockTime,
+        bytes32 _startingL2Output,
         uint256 _startingBlockNumber,
         uint256 _startingTimestamp,
-        uint256 _l2BlockTime,
         address _proposer,
         address _owner
     ) Semver(0, 0, 1) {
-        require(
```

Comment:
I feel like this require was to prevent config issues, a bad value here could cause the opnode to break

Reply:
It's in `initialize` now